### PR TITLE
Updated purchase request api documentation with suggested changes

### PR DESCRIPTION
**Pull request by harishrconcur**
Updated purchase request api documentation with suggested changes by Charlie Owen
1. Added getting started sections
2. Added PRE-RELEASE information
3. Added links to postman file for the API
4. Updated some of the API documentation based on latest changes

### CODE REVIEW

**Comment by charlieconcur** [comment]
We need a peer review of the changes.

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+    POST  /purchaserequest/v1/purchaserequests
+
+Create a Purchase Request based on provided header and line item details. If the request is valid it returns back a unique identifier to look at purchase request details and creates a purchase request in back ground.
+
+### <a name="8"></a>Parameters  
+
+|Name | Type | Format | Description
+|-----|------|--------|------------
+|`X_AUTH_TOKEN`|`string`|`header`|**Required**: Bearer Token that identifies the caller. This is the company JWT
```
Comment (ShanmugaprabhuRamadoss-concur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: Can we rename `X_AUTH_TOKEN` to `Authorization`

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    -d '{
+      "description" : "New office supplies",
+      "userLoginId" : "john.deo@concur",
+      "policyExternalId" : "po-external-id",
+      "currencyCode" : "USD",
+      "notesToSupplier" : "Office space request phase 1",
+      "comments" : "office supplies request",
+      "custom1" : "ADVT",
+
+          "lineItems" : [
```
Comment (ShanmugaprabhuRamadoss-concur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: We can remove the indent here.

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+    GET  /purchaserequest/v1/purchaserequests/{id}?mode=COMPACT
+
+Gets purchase request details. Currently only supported mode is COMPACT which returns basic info about the purchase request along with any exceptions if present.
+
+### <a name="8"></a>Parameters  
+
+|Name | Type | Format | Description
+|-----|------|--------|------------
+|`X_AUTH_TOKEN`|`string`|`header`|**Required**: Bearer Token that identifies the caller. This is the company JWT
```
Comment (ShanmugaprabhuRamadoss-concur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: Authorization header

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+### Response
+
+
+[Schema](#get_purchase_request_schema_response)
+
+- Example - Output
+ ```json
+     {
+        "purchaseRequestId" : "de9c0894-b807-6943-8e3f-49a707da3456",
+        "purchaseRequestNumber" : "PR100000",
```
Comment (ShanmugaprabhuRamadoss-concur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: our PR number never starts with alphas, so we can just have 1000

File: src/api-reference/invoice/purchase-request-endpoints.markdown [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+|`unitPrice`|`decimal`|-|**Required**: The unit price of the line item
+|`receiptType`|`string`|-|Accepted values are QUANTITY_RECEIPT or NONE. If a value is not provided the items it will default to NONE for SERVICES, or QUANTITY_RECEIPT for GOODS items based on purchase type.  If you are using Concur Receiving and need to enter Goods Receipts against the resulting PO lines use QUANTITY_RECEIPT
+|`neededByDate`|`date`|-|The date by which the purchase order must be fulfilled. Format: YYYY-MM-DD
+|`uoMCode`|`string`|-|Unit of Measure (UOM) code for the purchase request item. Accepted values are the UOM Codes setup in the Unit of Measure configuration. If no value is provided it will default a UOM based on the defaults for goods and services
+|`shipping`|`decimal`|-|The total shipping cost for the Item
+|`tax`|`decimal`|-|Tax amount that is associated with the line item
+|`supplierPartId`|`string`|-|An Id value that helps to identify the line item. This could be a value such as the vendor’s part number or even the manufacturer number
+|`url`|`string`|-|A valid URL related to the item. You can have multiple URL's per item
```
Comment (ShanmugaprabhuRamadoss-concur) [inline src/api-reference/invoice/purchase-request-endpoints.markdown]: url is an array of string

Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Fixed in f847a0bc

File: src/api-reference/invoice/purchase-request-endpoints.markdown (hunk 4 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Fixed in f847a0bc

File: src/api-reference/invoice/purchase-request-endpoints.markdown (hunk 3 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Fixed in f847a0bc

File: src/api-reference/invoice/purchase-request-endpoints.markdown (hunk 2 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Fixed in f847a0bc

File: src/api-reference/invoice/purchase-request-endpoints.markdown (hunk 1 above)
Comment (harishrconcur) [reply src/api-reference/invoice/purchase-request-endpoints.markdown]: Fixed in f847a0bc

**Pull request merged**